### ISSUE TITLE
Image captions not displayed without JS

### ISSUE THREAD

**Issue by jamesgorrie** [wellcometrust/wellcomecollection.org#4055]
Captions should be displayed without JS as they aren't JS specific.

**Comment by jamesgorrie** [comment]
My thoughts on this is we should probably not have all the JS that runs on the current `CaptionedImage`but rather only use run it when we need.

This would help with rendering concerns as well as any performance hits we're taking for re-drawing the images with JS.
